raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -49,6 +49,24 @@ import { makeAddress, makeHash } from '../utils';
 
 jest.mock('@ethersproject/providers');
 
+jest.mock('rxjs/fetch', () => ({
```

Comment:
@andrevmatos Is there a way to do this in the test locally? It's not nice to have to do this in the global namespace.

Reply:
Unfortunately, not; these mocks need to go in the global namespace, because they need to be mocked at `import` time (of the modules using the mocked values). Despite that, you should not mock `rxjs/fetch` directly, but instead the `globalThis.fetch` API, which is used by `fromFetch` under the hood. Give a look at `integration/mocks.ts` on how that can be done.

Reply:
@palango if you are done with these changes then please let me know everything looks fine, so i could approve it.

Reply:
What's the advantage of overwriting `globalThis.fetch`?

Reply:
Anyhow, fix is done.

Reply:
The advantage is to be able to interact/mock a more streamlined API (fetch), and also not depend if it's used through the `fromFetch` helper or directly (e.g. we use both in the SDK codebase)